src/main/java/com/github/onsdigital/babbage/api/endpoint/content/MaxAge.java

```diff
@@ -54,6 +56,12 @@ protected boolean verifyKey(String key) {
     }
 
     protected int getMaxAge(HttpServletRequest request) throws Exception {
+        if (appConfig().babbage().isLegacyCacheAPIEnabled()) {
+            LegacyProxyMaxAge legacyProxyMaxAge = new LegacyProxyMaxAge();
+            int maxAge = legacyProxyMaxAge.getMaxAgeValueFromLegacyProxy(request);
```

Comment:
You have created a very descriptive variable and method names, so it's clear what this returns, which means that you don't need to save it in a variable, you can just return it directly.